avocado_vt/vt_agent/src/avocado_vt/agent/managers/resource_backing_manager.py

```
@@ -0,0 +1,242 @@
+        """
+        Update the resource by its backing object.
+        """
+        r, o = 0, dict()
+        try:
+            backing = self.backings[backing_id]
+            pool_conn = self.pool_connections[backing.resource_pool_uuid]
+            handler = backing.get_update_handler(command)
+            ret = handler(pool_conn, arguments)
+            if ret:
+                o["out"] = ret
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)